libkbfs/journal_md_ops_test.go

```diff
@@ -129,12 +129,12 @@ func TestJournalMDOpsBasics(t *testing.T) {
 	id, head, err := mdOps.GetForHandle(ctx, h, Merged)
 	require.NoError(t, err)
 	require.NotEqual(t, tlf.NullID, id)
-	require.NotNil(t, head)
+	require.NotEqual(t, ImmutableRootMetadata{}, head)
```

Comment:
I agree with this change, but the check immediately below this already guarantees it's not an empty struct, right?